tests/testing_tools.py

```diff
@@ -0,0 +1,51 @@
+from datetime import date, timedelta
+import random
+from faker import Faker
+import csv
+from PIL import Image
+import os
+import shutil
+class TestingTools:
+    def test_date(self):
+        """test_date: creates an arbitrary date, 20 years in the past from today's date,
+           to create test files for, so as not to overwrite current work
+           ! if this code outlives 20 years of use I would be impressed"""
+        unit_date = date.today() - timedelta(days=365 * 20)
+        return str(unit_date)
+
+    def create_fake_dataset(self, num_records: int, path_list: list):
```

Comment:
create_fake_dataset: creates a random fake dataset with certain relevant real column names of any length, and at any path location.